pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/definition/tabular/TabularRowDef.java

```diff
@@ -2,15 +2,18 @@
 
 import com.google.gson.annotations.SerializedName;
 import lombok.Value;
-import org.broadinstitute.ddp.model.activity.definition.question.QuestionDef;
+import org.broadinstitute.ddp.model.activity.definition.QuestionBlockDef;
 
 import javax.validation.Valid;
 import javax.validation.constraints.NotEmpty;
+import java.util.ArrayList;
 import java.util.List;
 
 @Value
 public class TabularRowDef {
+
     @NotEmpty
     @SerializedName("questions")
-    List<@Valid QuestionDef> questions;
+    private List<@Valid QuestionBlockDef> questions = new ArrayList<>();
```

Comment:
No need to add `private` here since the class is annotated as Value. Lombok will generate it